SmartDeviceLink-iOS/SmartDeviceLink/SDLLocalization.h

```diff
@@ -0,0 +1,83 @@
+// SDLLocalization.h
+//
+
+
+#import <Foundation/Foundation.h>
+
+/**
+ Provides a simplified way to handle localization for a specific language that may not match the system language.
+ */
+@interface SDLLocalization : NSObject
+
+/** The language code used by this instance. This code can differ from initialization regarding fallback. */
+@property (readonly) NSString *language;
+
+/** The region code used by this instance. This code can differ from initialization regarding fallback. */
+@property (readonly) NSString *region;
```

Comment:
Can you give an example region code. And region codes as defined by whom? SDL (which I think uses an RFC)?